app/src/main/java/cm/aptoide/pt/home/Home.java

```diff
@@ -29,4 +33,17 @@ public Home(BundlesRepository bundlesRepository) {
   public boolean hasMore() {
     return bundlesRepository.hasMore();
   }
+
+  @NonNull private List<HomeBundle> addFakeSocialBundleTo(HomeBundlesModel homeBundlesModel) {
```

Comment:
Not being called

Reply:
moved the unused logic to the fake bundle data source.